packages/driver/package.json

```diff
@@ -54,6 +54,10 @@
     "moment": "2.24.0",
     "morgan": "1.9.1",
     "npm-install-version": "6.0.2",
```

Comment:
We can remove `npm-install-version` as a dependency now that we don't use it, right?